docs/book/src/user/quick-start.md

```diff
@@ -358,7 +361,21 @@ For more information about prerequisites, credentials management, or permissions
 {{#/tab }}
 {{#tab OpenStack}}
 
-Please visit the [OpenStack getting started guide].
+A ClusterAPI compatible image must be available in your OpenStack. For instructions on how to build a compatible image 
+see [image-builder](https://image-builder.sigs.k8s.io/capi/providers/openstack.html). 
```

Comment:
This is actually QEMU image for KVM. OpenStack supports many hypervisors: https://docs.openstack.org/nova/train/admin/arch.html#hypervisors .

It might be better to address VMware vSphere if you use VMware integrated OpenStack and ova image:
```
If you use VMware vSphere, see [Building images for vSphere](https://image-builder.sigs.k8s.io/capi/providers/vsphere.html)
```

Reply:
Thx for the hint. I added a comment although I'm using images which are converted from the output of a qemu build in our VIO right now and it works without any problems :)

Reply:
Thanks for your information. I am sorry I am not sure VIO usage :(